projects/kubernetes/python/README.md

```diff
@@ -0,0 +1,198 @@
+# Running Python Application on Kubernetes
+
+## Kubernetes is an open source platform that offers deployment, maintenance, and scaling features. It simplifies management of containerized Python applications while providing portability, extensibility, and self-healing capabilities.
+
+## Whether Python application are simple or complex, Kubernetes lets you efficiently deploy and scale them seamlessly rolling out new features while limiting resources to only those required.
+
+### Pre-requistes
+
+-Docker
+-Kubectl
+-Sample Python App
+
+## Docker is an open platform to build and ship distributed applications. To install Docker, follow the Docker official documentation.
+
+## To verify that Docker runs your system:
+
+```
+$ docker info
+Containers: 0
+Images: 289
+Storage Driver: aufs
+Root Dir: /var/lib/docker/aufs
+Dirs: 289
+Execution Driver: native-0.2
+Kernel Version: 3.16.0-4-amd64
+Operating System: Debian GNU/Linux 8 (jessie)
+WARNING: No memory limit support
+WARNING: No swap limit support
+```
+
+### kubectl is a command-line interface for executing commands against a Kubernetes cluster. Run the shell script below to install kubectl
+
+```
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+
+```
+### Deploying to Kubernetes requires a containerized application. Let's review containerizing Python applications.
+
+### Containerization at a glance
+### Containerization involves enclosing an application in a container with its own operating system. This full machine virtualization option has the advantage of being able to run an application on any machine without concerns about dependencies.
+
+### Create a Python container image
+
+## To create these images, we will use Docker, which enables us to deploy applications inside isolated Linux software containers. Docker is able to automatically build images using instructions from a Docker file.
```

Comment:
Deploying to Kubernetes requires a containerized application. Let's review containerizing Python applications.

### Containerization at a glance
Containerization involves enclosing an application in a container with its own operating system. This full machine virtualization option has the advantage of being able to run an application on any machine without concerns about dependencies.

### Create a Python container image

To create these images, we will use Docker, which enables us to deploy applications inside isolated Linux software containers. Docker is able to automatically build images using instructions from a Docker file.